src/pages/all-on-one-mini-man/all-on-one-mini-man.ts

```diff
@@ -0,0 +1,105 @@
+import { HazardRecorderProvider } from './../../providers/hazard-recorder/hazard-recorder';
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { FaultsScorecardProvider } from '../../providers/faults-scorecard/faults-scorecard';
+import { FaultStoreProvider } from '../../providers/fault-store/fault-store';
+
+/**
+ * Generated class for the AllOnOnePage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-all-on-one-mini-man',
+  templateUrl: 'all-on-one-mini-man.html',
+})
+export class AllOnOneMiniManPage {
+  isDButtonPressed = false;
+  isSButtonPressed = false;
+
+  showManoeuvres: boolean = false;
```

Comment:
Don't need to explicitly type these variables, type can be inferred from the value